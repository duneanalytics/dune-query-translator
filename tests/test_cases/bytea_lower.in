select lower('0xfoo'), 'hello'
