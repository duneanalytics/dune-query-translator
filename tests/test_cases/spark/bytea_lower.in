select lower('0xdeadbeef'), 'hello'
