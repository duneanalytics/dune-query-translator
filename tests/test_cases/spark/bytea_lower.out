SELECT
  0xdeadbeef,
  'hello'